src/main.cpp

```diff
@@ -981,6 +981,36 @@ bool ContextualCheckTransaction(
             // https://zips.z.cash/zip-0213#specification
             uint256 ovk;
             for (const OutputDescription &output : tx.vShieldedOutput) {
+                bool zip_212_enabled;
+                libzcash::SaplingPaymentAddress zaddr;
+                CAmount value;
+
+            // 5.4.2 EoS height is 2121024.
```

Comment:
The indentation is weird here. (It's fine to indent the unchanged lines 1014-1039.)

Reply:
Doing this introduces unnecessary noise, as we'd simply be un-indenting them again in the future PR. The negative indent affects four lines and is temporary.